sdk/core/azure-core-tracing-opentelemetry/azure/core/tracing/ext/opentelemetry_span/__init__.py

```diff
@@ -135,9 +134,8 @@ def __exit__(self, exception_type, exception_value, traceback):
             self._current_ctxt_manager = None
 
     def start(self):
-        # type: () -> None
-        """Set the start time for a span."""
-        self.span_instance.start()
+        # we still need start for the on_request on distributed_tracing policy.
```

Comment:
Please remove this comment, the reason we have "start"is because it's part of the ABC this class implements, it has nothing to do any implementation that we know exists :)

If you want to justify the "pass", then do something like "span are automatically started at their creation with OT" or something like that